project_milestone/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+project_milestone_user,Project Milestone User,model_project_milestone,project.group_project_user,1,1,1,1

```

Comment:
IMO User should have read access only, and write access should be reserved for Manager (Tasks are like this, right?)

Reply:
You are correct, i've updated the security to include the manager group and reset the user access.